core/src/main/scala/com/evolutiongaming/kafka/flow/ConsumerFlow.scala

```diff
@@ -84,11 +84,10 @@ object ConsumerFlow {
     def poll = {
       val flowList = flows.toList // optimization, execute toList once instead of on each `consumer.poll`
       consumer.poll(config.pollTimeout) flatTap { consumerRecords =>
-        flowList traverse { case (topic, flow) =>
-          val topicRecords = consumerRecords.values filter { case (partition, _) =>
-            partition.topic == topic
-          }
-          flow(ConsumerRecords(topicRecords))
+        val recordsByTopic = consumerRecords.values.groupBy(_._1.topic)
```

Comment:
```
consumerRecords
.values
.groupBy(_._1.topic)
.parFoldMapA { (topic, records) +> } 
```

please, we do not need use `flowList` here, right?

Reply:
we still need to call `apply` on flows even when we haven't received record for the topic, as we need to trigger timers for it

Reply:
than I propose the only change to be done: replace `traverse` with `parFoldMapA`, you'd better leave `filter` inside of parallelised snippet

Reply:
the filter inside is not going to be parallelised, only the IO will be. the function itself is called sequentially

Reply:
it's more complex than that, you can make a test to check it out.